src/Captcha.Net/CaptchaGenerator.cs

```diff
@@ -1,16 +1,52 @@
 ﻿using System;
 using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Captcha.Net
 {
+    public static class ShuffleList
+    {
+        private static readonly Random random = new();
+
+        public static void Shuffle<T>(this IList<T> list)
+        {
+            var n = list.Count;
+            while (n > 1)
+            {
+                n--;
+                var k = random.Next(n + 1);
+                (list[k], list[n]) = (list[n], list[k]);
+            }
+        }
+    }
+
     public class CaptchaGenerator
     {
-        private static char[] CodeLetters = new char[] { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9' };
-        private static ConcurrentDictionary<string, Captcha> CaptchaKeeper = new ConcurrentDictionary<string, Captcha>();
+        private static char[] _codeLetters = Array.Empty<char>();
+        private static readonly ConcurrentDictionary<string, Captcha> CaptchaKeeper = new();
 
-        public string GenerateCaptchaCode(int keyDigit = 4)
+        public string GenerateCaptchaCode(int keyDigit = 6)
         {
-            return Extensions.GetUniqueKey(keyDigit, CodeLetters);
+            var lowercaseLatinLetters = Enumerable.Range('a', 'z' - 'a' + 1).Select(c => (char)c);
```

Comment:
_codeLetters must be customizable. maybe the client just needs the numerical letters.